devices/osram.js

```diff
@@ -437,19 +444,20 @@ module.exports = [
         model: '4062172044776_4',
         vendor: 'OSRAM',
         description: 'Zigbee 3.0 DALI CONV LI dimmer for DALI-based luminaires (with two devices and pushbutton)',
+        fromZigbee: [...extend.ledvance.light_onoff_brightness({noConfigure: true}).fromZigbee,
+            fz.command_toggle, fz.command_move, fz.command_stop],
         extend: extend.ledvance.light_onoff_brightness({noConfigure: true}),
-        exposes: [e.light_brightness().withEndpoint('l1'), e.light_brightness().withEndpoint('l2')],
+        exposes: [e.action(['toggle', 'brightness_move_up', 'brightness_move_down', 'brightness_stop']),
+            e.light_brightness().withEndpoint('l1'), e.light_brightness().withEndpoint('l2')],
+        ota: ota.zigbeeOTA,
         endpoint: (device) => {
-            return {'l1': 10, 'l2': 11};
+            return {'l1': 10, 'l2': 11, 's1': 25};
```

Comment:
Why was `'s1': 25` added here?

Reply:
If I don't do this, I get a "toggle_undefined" in the "action".  Unfortunately I don't know how to do this differently.

Reply:
Can you check if the device still works fine when removing the `meta: {multiEndpoint: true},` line?

Reply:
Yes it still works. I can remove this. What does this function do exactly? I thought this is required?

Reply:
Ah wait, it is needed. Can you update the actions to have the `_s1` postfix?